### ISSUE TITLE
Social login templates broken

### ISSUE THREAD

**Issue by giohappy** [GeoNode/geonode-mapstore-client#827]
Apparently the templates provided by the new client break django-allauth social login templates.
The login pages look empty, but the HTML code shows the form login elements.

django-allauth templates (e.g. [login](https://github.com/pennersr/django-allauth/blob/0.48.0/allauth/templates/account/login.html#L1)) extend the [account/base.html](https://github.com/GeoNode/geonode/blob/675a0490a2c457b99d44d89402937b67a93c8924/geonode/templates/account/base.html#L1) template, which is overridden by GeoNode. 
Probably the overrides from the client to GeoNode base templates break the layout of social logins.

The issue has been reported in https://github.com/GeoNode/geonode/issues/8806

![image](https://user-images.githubusercontent.com/571129/153855546-01225532-f9ae-4b71-8913-24339bc486b1.png)

**Comment by allyoucanmap** [comment]
@giohappy What's the content we expect in this page? I tested locally but I was not able to render the content. The modal you are highlighting is the login one that could be triggered from the legacy page but it does not contain new fields.

I did following tests:

- disable the custom base.html from the geonode-mapstore-client but the result is the same

![image](https://user-images.githubusercontent.com/19175505/154040332-bccd9c53-9401-4253-a84a-007c2d3a337e.png)

- the node you are highlighting is the login modal that does not contains additional fields so not related to the social account

![image](https://user-images.githubusercontent.com/19175505/154040453-63210c32-ffa0-4ba1-8590-989073e13196.png)

- tested on **3.3.x** and the page is redirect to the google account authentication as soon as we land on `/account/google/login
` but this is not happening on master. Which part of the code is redirecting to the google page on 3.3.x?

![image](https://user-images.githubusercontent.com/19175505/154041597-81ffe72f-d7af-4a5c-a462-eed1f49513b4.png)
